static/bidder-params/ix.json

```diff
@@ -4,11 +4,21 @@
   "description": "A schema which validates params accepted by the Ix adapter",
   "type": "object",
   "properties": {
+    "siteid": {
+      "type": "string",
+      "minLength": 1,
+      "description": "An ID which identifies the site selling the impression"
+    },
     "siteId": {
       "type": "string",
       "minLength": 1,
       "description": "An ID which identifies the site selling the impression"
     },
+    "siteID": {
+      "type": "string",
+      "minLength": 1,
+      "description": "An ID which identifies the site selling the impression"
+    },
```

Comment:
Some hosting companies, such as Xandr, provide a user interface to publishers based on this json schema. To help the UI understand which field to ask the publisher to provide in a case such as this (multiple fields differing only by case) can you please add the word "preferred" to the description of the format you'd like to be the default?

Please see [Criteo's json schema](https://github.com/prebid/prebid-server/blob/master/static/bidder-params/criteo.json) as an example.

Thank you.

Reply:
done.